dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -1139,23 +1139,41 @@ def get_record_with_sql(select_attribute, table_name, selector)
   #
 
   test "clears form_data from pd_teachercon1819_registrations" do
-    registration = create :pd_teachercon1819_registration
-    refute_equal '{}', registration.form_data
+    teacher = create :teacher
+    created_at = Time.now.to_s(:db)
+    ActiveRecord::Base.connection.exec_query(
+      <<-SQL
+        INSERT INTO `pd_teachercon1819_registrations` (user_id, form_data, created_at, updated_at)
+        VALUES (#{teacher.id}, '{\"country\": \"USA\"}', '#{created_at}', '#{created_at}')
+    SQL
+    )
+
+    registration = get_record_with_sql("form_data", "pd_teachercon1819_registrations", {'created_at' => "'#{created_at}'"})
+    refute_empty registration["form_data"]
 
-    purge_user registration.user
+    purge_user teacher
 
-    registration.reload
-    assert_equal '{}', registration.form_data
+    registration = get_record_with_sql("form_data", "pd_teachercon1819_registrations", {'created_at' => "'#{created_at}'"})
+    assert_empty registration["form_data"]
   end
 
   test "clears user_id from pd_teachercon1819_registrations" do
-    registration = create :pd_teachercon1819_registration
-    refute_nil registration.user_id
+    teacher = create :teacher
+    created_at = Time.now.to_s(:db)
+    ActiveRecord::Base.connection.exec_query(
+      <<-SQL
+        INSERT INTO `pd_teachercon1819_registrations` (user_id, form_data, created_at, updated_at)
+        VALUES (#{teacher.id}, '{\"country\": \"USA\"}', '#{created_at}', '#{created_at}')
+    SQL
+    )
 
-    purge_user registration.user
+    registration = get_record_with_sql("user_id", "pd_teachercon1819_registrations", {'created_at' => "'#{created_at}'"})
+    refute_nil registration["user_id"]
+
+    purge_user teacher
```

Comment:
one other idea here is to simply grab `registration.id` before `purge_user`.

Reply:
Ah, that's clever. I'm going to go with not wiping the user id to be consistent with what we do with other models –– I was hesitant to change behavior, but I feel better with your recommendation :). Thank you!